front-end/src/app/reports/transactions/transaction-input/transaction-input.component.html

```diff
@@ -27,12 +27,21 @@
 
 @if (transaction?.contact_1) {
   @switch (entityType) {
-    @case (ContactTypes.ORGANIZATION || ContactTypes.COMMITTEE) {
```

Comment:
Unfortunately, we can't use @case to match against multiple values at one time.  In this situation, the switch was comparing the value of `entityType` to `true` which is the result of the expression `(ORGANIZATION || COMMITTEE)`